virt/virt-2-5-release-notes.adoc

```diff
@@ -232,3 +232,6 @@ To unlock the account, log in to the Manager and enter the following command:
 ----
 $ ovirt-aaa-jdbc-tool user unlock admin
 ----
+
+//New known issue for 2.5.8
+* If you are logged in to the {product-title} as a user with `basic-user` privileges, trying to retrieve guest agent information by running `virtctl guestosinfo <vmi_name>` fails. As a workaround, you can fetch a subset of the guest agent data by using the `oc describe vmi` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2000464[*BZ#2000464*])
```

Comment:
s/the {product-title}/the {product-title} cluster

s/trying to retrieve/retrieving

s/data by using the/data by running the

Reply:
Fixed